ros/api/v1/hosts.py

```diff
@@ -518,10 +576,16 @@ def get(self):
                     "under_pressure": cpu_states_dict['under_pressure']
                 }
             },
+            "instance_types_highlights": {
+                "current": highlights_instance_types(current_performance_profiles, 'current'),
+                "suggested": highlights_instance_types(current_performance_profiles, 'suggested'),
```

Comment:
I think we should not use `current_performance_profiles` for suggested system. For suggested we should get the systems from `performance_profile_history` table.
Reason - If you check the executive report these systems are showed under `Most suggested instance types (yesterday)`. suppose user downloaded the report and he saw suggestion is to move to t2.medium from t2.small, then user added few more systems to ros say t2.large and t2.medium systems and downloaded the executive report again(on the same day). Now the section of `Most suggested instance types (yesterday)` will differ in earlier report and the new reported. Newly downloaded report will show new suggestion under **yesterday** section but hey system was created and reported today. 
So for suggested please select system from performance_profile_history table where report_date == current_date - 1.

Reply:
As discussed, we would like to populate the `current` and  `suggested` section from the same rule hit to show the latest of suggestions per performance_profile.

The same don't exist in the historical table atm.

Reply:
resolving this conversation considering we need to remove **yesterday** keyword from the executive report frontend.